packages/airnode-examples/integrations/config-utils.ts

```diff
@@ -1,9 +1,15 @@
 import { writeFileSync } from 'fs';
 import { join } from 'path';
 import { Config, LocalOrCloudProvider } from '@api3/airnode-node';
-import { cliPrint, readIntegrationInfo } from '../src';
+import { cliPrint, getDeployedContract, readChainId, readIntegrationInfo } from '../src';
+
+export const createCloudProviderConfiguration = (generateExampleFile: boolean): LocalOrCloudProvider => {
```

Comment:
This was previously a bug - the function always looked into integration-info.json for values.